src/main/java/org/broadinstitute/dsm/db/dto/kit/BSPKitDto.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.dsm.db.dto.kit;
+
+import lombok.Getter;
+
+@Getter
+public class BSPKitDto {
+    private final String baseUrl;
+    private final String bspSampleId;
+    private final String bspParticipantId;
+    private final String instanceName;
+    private final String bspOrganism;
+    private final String bspCollection;
+    private final String ddpParticipantId;
+    private String bspMaterialType;
+    private String bspReceptacleType;
+    private boolean hasParticipantNotifications;
+    private String participantExitId;
+    private String deactivationDate;
+    private String notificationRecipient;
+
+    public BSPKitDto(String instanceName,
```

Comment:
I prefer to use `Builder` design pattern when it comes such complex object construction, `Builder` has many benefits including much more understanding while creating of object since it follows `withBaseUrl` standard and there is a less chance that different value will be passed to different field, except that `Builder` provides simplicity while testing, since you don't have to use constructor which has such many parameters.